eng/common/scripts/Prepare-Release.ps1

```diff
@@ -0,0 +1,108 @@
+#Requires -Version 6.0
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$PackageName,
+    [string]$ServiceDirectory,
+    [string]$ReleaseDate, # Pass Date in the form MM/dd/yyyy"
+    [string]$BuildType # For Java
+)
+
+. ${PSScriptRoot}\common.ps1
+
+function Get-ReleaseDay($baseDate)
+{
+    # Find first friday
+    while ($baseDate.DayOfWeek -ne 5)
+    {
+        $baseDate = $baseDate.AddDays(1)
+    }
+    
+    # Go to Tuesday
+    $baseDate = $baseDate.AddDays(4)
+
+    return $baseDate;
+}
+
+$ErrorPreference = 'Stop'
+
+$packageProperties = Get-PkgProperties -PackageName $PackageName -ServiceDirectory $serviceDirectory
+
+Write-Host "Source directory [ $serviceDirectory ]"
+
+if (!$ReleaseDate)
+{
+    $currentDate = Get-Date
+    $thisMonthReleaseDate = Get-ReleaseDay((Get-Date -Day 1));
+    $nextMonthReleaseDate = Get-ReleaseDay((Get-Date -Day 1).AddMonths(1));
+
+    if ($thisMonthReleaseDate -ge $currentDate)
+    {
+        # On track for this month release
+        $ParsedReleaseDate = $thisMonthReleaseDate
+    }
+    elseif ($currentDate.Day -lt 15)
+    {
+        # Catching up to this month release
+        $ParsedReleaseDate = $currentDate
+    }
+    else 
+    {
+        # Next month release
+        $ParsedReleaseDate = $nextMonthReleaseDate
+    }
+}
+else
+{
+    $ParsedReleaseDate = ([datetime]$ReleaseDate, 'MM/dd/yyyy', [Globalization.CultureInfo]::InvariantCulture)
+}
+
+$releaseDateString = $ParsedReleaseDate.ToString("MM/dd/yyyy")
+$month = $ParsedReleaseDate.ToString("MMMM")
+
+Write-Host
+Write-Host "Assuming release is in $month with release date $releaseDateString" -ForegroundColor Green
+
+$currentProjectVersion = $packageProperties.Version
+
+$newVersion = Read-Host -Prompt "Input the new version, or press Enter to use use current project version '$currentProjectVersion'"
+
+if (!$newVersion)
+{
+    $newVersion = $currentProjectVersion;
+}
+
+$newVersionParsed = [AzureEngSemanticVersion]::ParseVersionString($newVersion)
+if ($null -eq $newVersionParsed)
+{
+    Write-Error "Invalid version $newVersion. Version must follow standard SemVer rules, see https://aka.ms/azsdk/engsys/packageversioning"
```

Comment:
Can you have this link point to https://azure.github.io/azure-sdk/policies_releases.html#package-versioning.